tests/testthat/test-tabyl.R

```diff
@@ -391,7 +395,7 @@ test_that("3-way tabyl with 3rd var factor is listed in right order, #250", {
   z$cyl[32] <- NA
   expect_equal(names(tabyl(z, am, gear, cyl)), c("4", "8", "6", "NA_"))
   expect_equal(names(tabyl(z, am, gear, cyl, show_na = FALSE)), c("4", "8", "6"))
-  z <- z %>% dplyr::filter(! cyl %in% "4")
+  z <- z %>% dplyr::filter(!cyl %in% "4")
```

Comment:
Like here's one of many pipelines on one line that it left alone

Reply:
according to the tidyverse style guide, 1 pipe is okay on a single line https://style.tidyverse.org/pipes.html#short-pipes

Reply:
ah that was it! Thank you